### DISCUSSION TITLE
Allow deleting workflow runs from run's page

### DISCUSSION THREAD

**Discussion by janosh** [community/community#37932] [Actions]
Would be convenient if workflow runs could be deleted from the run's page:

![Screenshot 2022-10-31 at 17 44 45](https://user-images.githubusercontent.com/30958850/199135111-622ee649-e7d9-4a15-bef1-acf190d72efc.png)

This currently requires going to the overview page:

![Screenshot 2022-10-31 at 17 47 15](https://user-images.githubusercontent.com/30958850/199135180-652a296a-290d-44c1-a7a6-60e0d197d43f.png)

**Comment by DSDmark** [comment]
I guess it's because of **event** when its trigger, its need to workflow. I think that way is necessary
